src/components/shared/SourceIcon.css

```diff
@@ -19,6 +19,11 @@
   fill: var(--theme-textbox-box-shadow);
 }
 
+.source-icon.angular {
+  background: url(/images/angular.svg) 1px 1px no-repeat;
+  background-size: 13px 13px;
+}
+
```

Comment:
lets also add Vue

https://github.com/devtools-html/debugger.html/blob/master/src/workers/parser/frameworks.js#L17-L19

we might need a good way to safeguard this from future frameworks down the road...